src/extension/src/scripts/run-tests.ts

```diff
@@ -0,0 +1,150 @@
+let child_process = require("child_process");
+const path = require("path");
+const fs = require("fs");
+
+const testFolder = path.join(__dirname, "..", "..", "src", "template_test");
+console.log(testFolder);
+let files: string[] = [];
+
+//Recursively delete all generated projects in template_test
+let deleteFolderRecursive = function(pathname: string) {
+  let files = [];
+  if (fs.existsSync(pathname)) {
+    files = fs.readdirSync(pathname);
+    files.forEach(function(file: string, index: number) {
+      var curPath = path.join(pathname, file);
+      if (fs.lstatSync(curPath).isDirectory()) {
+        deleteFolderRecursive(curPath);
+      } else {
+        // delete file
+        fs.unlinkSync(curPath);
+      }
+    });
+    fs.rmdirSync(pathname);
+    console.log(`deleted ${pathname}`);
+  }
+};
+
+fs.readdirSync(testFolder).forEach((file: string) => {
+  files.push(file.toString());
+});
+
+function kill(pid: any) {
+  try {
+    child_process.execSync("taskkill /PID " + pid + " /T /F", function(
```

Comment:
mac and linux?

Reply:
This test will run just on CI VM, so I don't think we have to handle all the platforms, right?

Reply:
Oh so we'll never run it locally? If not I wish there was more indication on this via the folder structure.

Reply:
Yeah, it's only going to run on powershell so it'll be fine